src/token/erc721/interface.cairo

```diff
@@ -137,57 +137,3 @@ trait IERC721ReceiverCamel<TState> {
         data: Span<felt252>
     ) -> felt252;
 }
-
-//
-// ERC721 ABI
```

Comment:
why was this removed? is it related to merging https://github.com/OpenZeppelin/cairo-contracts/pull/761/files?

Reply:
Yes, exactly